pkg/gen/typegen.go

```diff
@@ -255,20 +255,33 @@ func createGroups(definitionsJSON map[string]interface{}, generatorType gentype)
 				return linq.From([]KindConfig{})
 			}
 
-			// Make fully-qualified group name.
-			fqGroup := d.gvk.Group
+			// Make fully-qualified and "default" GroupVersion. The "default" GV is the `apiVersion` that
+			// appears when writing Kubernetes YAML (e.g., `v1` instead of `core/v1`), while the
+			// fully-qualified version is the "official" GV (e.g., `core/v1` instead of `v1` or
+			// `admissionregistration.k8s.io/v1alpha1` instead of `admissionregistration/v1alpha1`).
+			defaultGroupVersion := d.gvk.Group
```

Comment:
Unless I am misreading, `defaultGroupVersion` is appears to be reassigned on all branches below (unless I'm misreading), so should it be `var defaultGroupVersion string`, like `fqGroupVersion`?

Reply:
That's right, this was fixed in a subsequent push.